doc/sphinx/Pacemaker_Development/components.rst

```diff
@@ -169,18 +169,18 @@ messaging layer callback, which calls:
 
   * ``handle_reply()``, which calls
 
-    * ``process_remote_stonith_exec()``, which calls either
-      ``call_remote_stonith()`` (to retry a failed operation, or try the next
-       device in a topology is appropriate, which issues a new
+    * ``fenced_process_fencing_reply()``, which calls either
+      ``request_peer_fencing()`` (to retry a failed operation, or try the next
+      device in a topology is appropriate, which issues a new
```

Comment:
guess there is something missing here.
maybe "... device in a topology that is appropriate ..." or to make clean that
the device has to be appropriate "... next appropriate device in a topology ..." ...

Reply:
I think that was supposed to be "as appropriate", meaning "next device if any"

Reply:
or "if appropriate"